src/create-client.ts

```diff
@@ -46,6 +46,7 @@ export async function createClient(
       return true;
     },
     onRedirectCallback = (_: RedirectParams) => {},
+    onRefresh: _onRefresh,
```

Comment:
i don't really know why we do this `_` stuff for some of these options, but i guess i'm going to stick with it.

Reply:
Seems like a "this is private" convention.

Part of me thinks this code is leaving many hints that it should be rewritten as a class (where we could make these TypeScript `private` or even runtime private with `#`).

But keeping with the existing convention for now makes sense.

Reply:
yeah i've had the same thought about moving this into a proper class.